packages/client/src/app/pages/portfolio/blogs/blogs.component.ts

```diff
@@ -0,0 +1,162 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { FormGroup, FormControl } from '@angular/forms';
+
+import { FrontendUser } from '@pulp-fiction/models/users';
+import { Blog, SetPublishStatus } from '@pulp-fiction/models/blogs';
+import { PaginateResult } from '@pulp-fiction/models/util';
+import { AuthService } from '../../../services/auth';
+import { Constants, Title } from '../../../shared';
+import { PortBlogs } from '../../../models/site';
+import { MatDialog } from '@angular/material/dialog';
+import { CreateBlogComponent, EditBlogComponent, PreviewBlogComponent } from '../../../components/modals/blogs';
+import { BlogsService } from '../../../services/content';
+
+@Component({
+    selector: 'port-blogs',
+    templateUrl: './blogs.component.html',
+    styleUrls: ['./blogs.component.less']
+})
+export class BlogsComponent implements OnInit {
+    currentUser: FrontendUser;
+    portUser: FrontendUser;
+    blogsData: PaginateResult<Blog>
+    userBlogsData: PaginateResult<Blog>
+
+    pageNum = 1;
+    listView = false;
+
+    searchBlogs = new FormGroup({
+        query: new FormControl('')
+    });
+
+    constructor(private route: ActivatedRoute, private router: Router, private authService: AuthService, private dialog: MatDialog, private blogsService: BlogsService) {
+        this.authService.currUser.subscribe(x => {
+            this.currentUser = x;
+        });
+    }
+
+    ngOnInit(): void {
+        this.portUser = this.route.parent.snapshot.data.portData as FrontendUser;
+        Title.setThreePartTitle(this.portUser.username, Constants.BLOGS);
+
+        this.route.data.subscribe(data => {
+            const feedData = data.feedData as PortBlogs;
+            this.blogsData = feedData.blogs;
+            this.userBlogsData = feedData.userBlogs;
+        });
+    }
+
+    /**
+     * Handles page changing
+     * 
+     * @param event The new page
+     */
+    onPageChange(event: number) {
+        this.router.navigate([], {relativeTo: this.route, queryParams: {page: event}, queryParamsHandling: 'merge'});
+        this.pageNum = event;
+    }
+
+    /**
+     * Checks to see if the currently logged in user is the same as the user who owns this portfolio.
+     */
+    currentUserIsSame() {
+        if (this.currentUser) {
+            if (this.portUser) {
+                if (this.currentUser._id === this.portUser._id) {
+                    return true;
+                } else {
+                return false;
+                }
+            }
+        }
+    }
+
+    /**
+     * Switches the view
+     */
```

Comment:
Personal opinion time: I think these kinds of comments are actually harmful, i.e. "fooBar: foos the bar". It doesn't explain anything and just frustrates. Unless the documentation can explain a _why_ or a _how_, it doesn't need to exist. 

In this instance, the questions I'd probably ask are things like "_what_ view? Switches from what? To what?" Like, it's clear if you're reading the code, but that's not the point of a docstring comment.